Sources/Utils/POSIX/Stringify.cpp

```diff
@@ -170,6 +171,31 @@ char const *Stringify::Errno(int error) {
   }
 }
 
+char const *Stringify::WaitStatus(int status) {
+#define DO_WAIT_MSG(MESSAGE, VALUE)                                    \
+  do {                                                                 \
+    static ATT_TLS char tmp[30];                                       \
+    ds2::Utils::SNPrintf(tmp, sizeof(tmp), "%s: %s", MESSAGE, VALUE);  \
+    return tmp;                                                        \
+  } while (0);
```

Comment:
Nit (no need to change your PR): you shouldn't have a semicolon at the end of this line, because you already use `DO_WAIT_MSG()` as if it were a function (you add a semicolon at time of use).